src/test-unit/query-string.spec.js

```diff
@@ -0,0 +1,114 @@
+import { parse, stringify } from '../lib/query-string';
+
+describe('queryString', () => {
+    it('query string starting with a `?`', () => {
+        expect(parse('?foo=bar')).toEqual({foo: 'bar'});
+    });
+
+    it('query strings starting with a `#`', () => {
+        expect(parse('#foo=bar')).toEqual({foo: 'bar'});
+    });
+
+    it('query strings starting with a `&`', () => {
+        expect(parse('&foo=bar&foo=baz')).toEqual({foo: ['bar', 'baz']});
+    });
+
```

Comment:
Let’s add a test for a combination of “#” and “?”

Reply:
Those are two first in the beginning

Reply:
@jairo-bc I meant a string like this "'?foo=bar&a=b#foo=baz". Should hash parameters replace query parameters, or join them?

Reply:
join